src/AsyncEventSource.h

```diff
@@ -68,7 +68,8 @@ class AsyncEventSourceMessage {
   public:
     AsyncEventSourceMessage(const char * data, size_t len);
     ~AsyncEventSourceMessage();
-    size_t ack(size_t len, uint32_t time __attribute__((unused)));
+    size_t ack(size_t len);
```

Comment:
Since this is forked all over the place, I'll leave this signature and add a second one instead.